### DISCUSSION TITLE
¿How to start a thread?

### DISCUSSION THREAD

**Discussion by Eloscur** [community/community#21517] [Other Feature Feedback, Questions, & Ideas]
<p>Hello, i am new here and i got a lot of important questions about Termux as i haven’t found answers to them searching around i think it will help a lot a topic about them, so i’d like to know where i must start the thread about Termux.</p>

**Comment by Wabri** [comment, marked as answer]
<p>Hi <a class="mention" href="/eloscur">@eloscur</a>,</p>
<p>Welcome to the Github Community Forum!</p>
<p>For questions about your project or general discussions you can use the <a href="/t5/Programming-Help-and-Discussion/bd-p/programming">Programming-Help-and-Discussion</a>.</p>
<p>-Gabriele-</p>
<p>Mark helpfull posts with<span> </span>Accept as Solution<span> </span>to help other users locate important info. Don’t forget to give<span> </span>Kudos<span> </span>for great contents!</p>

**Comment by ss2131627** [comment]
<p>Thanks man,</p>
<p> I was looking for the same.</p>

**Comment by MChevy422** [comment]
<p>Hello <a class="mention" href="/eloscur">@eloscur</a> </p>
<p>This post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our Community organized so users can more easily find information.</p>
<p>As you’ll notice and as <a class="mention" href="/wabri">@wabri</a> mentioned, your topic is now in the’Programming Help &amp; Discussion’ board. No action is needed on your part; you can continue the conversation as normal here.</p>
<p>Let me know if you have any other questions or if I can help with anything else.</p>
<p>Greatly appreciated,</p>
<p><strong>-MChevy422</strong></p>
